Packs/SplunkPy/ReleaseNotes/3_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### SplunkPy
+- Added a new **Splunk Status Labels to Mirror** instance configuration option to specify specific status labels to mirror on XSOAR if closed on Splunk.
```

Comment:
Added a new *Splunk Status Labels to Mirror* instance configuration option to specify specific status labels to mirror in Cortex XSOAR if closed in Splunk.